src/pages/Menu.js

```diff
@@ -0,0 +1,240 @@
+    
+    const fontLink = document.createElement('link');
+    fontLink.rel = 'stylesheet';
+    fontLink.href = 'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap';
+    document.head.appendChild(fontLink);
+  }, []);
+
+  const products = [
+    {
+      id: 'burger-dreams',
+      name: 'Burger Dreams',
+      price: 9.20,
+      image: burgerDreams,
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.'
+    },
+    {
+      id: 'burger-waldo',
+      name: 'Burger Waldo',
+      price: 10.00,
+      image: burgerWaldo,
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.'
+    },
+    {
+      id: 'burger-cali',
+      name: 'Burger Cali',
+      price: 8.00,
+      image: burgerCali,
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.'
+    },
+    {
+      id: 'burger-bacon-buddy',
+      name: 'Burger Bacon Buddy',
+      price: 9.99,
+      image: burgerBaconBuddy,
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.'
+    },
+    {
+      id: 'burger-spicy',
+      name: 'Burger Spicy',
+      price: 9.20,
+      image: burgerSpicy,
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.'
+    },
+    {
+      id: 'burger-classic',
+      name: 'Burger Classic',
+      price: 8.00,
+      image: burgerClassic,
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.'
+    }
+  ];
+
+  return (
+    <>
+      <Header />
+      
+      <div className="header-section" style={{backgroundColor: '#f5f9fc', padding: '50px 0', clipPath: 'polygon(0.125rem 4.75rem, 100% 0%, 100% 100%, 0% 100%)'}}>
+        <div className="container-flex">
+          <div className="title-wrap-centre" style={{width: '100%', maxWidth: '700px', margin: '0 auto'}}>
+            <h1 
+              style={{
+                fontFamily: 'Inter, sans-serif',
+                color: '#35B8BE',
+                fontSize: '64px',
+                fontWeight: '400',
+                marginBottom: '20px',
+                textAlign: 'center',
+                lineHeight: '1.2'
+              }}
+            >
+              Browse our menu
+            </h1>
+            <p style={{
+              fontFamily: 'Inter, sans-serif',
+              fontSize: '16px', 
+              marginBottom: '30px', 
+              color: '#546285', 
+              textAlign: 'center',
+              lineHeight: '1.6'
+            }}>
+                           Use our menu to place an order online, or <span 
+                style={{ 
+                  color: '#35B8BE',
+                  position: 'relative',
+                  cursor: 'pointer'
+                }}
+                onMouseEnter={(e) => {
+                  const tooltip = document.createElement('div');
```

Comment:
that whole part should be a react component, generally there is not need to do document.createElement and other methods when you are using react